Automattic/jetpack
# E2E Tests: Fix tests in CI

Tests were failing mostly because of the new, unexpected _browser_ error. For some reason Chromium (only in headless mode?) thinks that Yahoo analytics script that we use on wordpress.com has some syntax error. Because of this error - browser was closing unexpectedly (since jest-puppeteer exits browser on error by default). 

#### Changes proposed in this Pull Request:

* Disable `exitOnPageError` for Puppeteer / Chromium
* Update some e2e packages
* add additional stuff to CI configuration step
* Update checkout page flow to use new iframed inputs

#### Testing instructions:






* You can try to run tests locally, just follow the instructions in Readme
* Check CI. it's green again! https://travis-ci.org/Automattic/jetpack/jobs/571903982

#### Proposed changelog entry for your changes:

* N/A
